src/main/java/info/ronjenkins/slf4bukkit/ColorMapperFactory.java

```diff
@@ -0,0 +1,23 @@
+package info.ronjenkins.slf4bukkit;
+
+/**
+ * Creates {@code ColorMapper} instances.
+ *
+ * @see ColorMapper
+ */
+public final class ColorMapperFactory {
+
+  /**
+   * Creates an new {@code ColorMapper} instance.
+   *
+   * @return a new instance
+   */
+  public static ColorMapper create() {
+    try {
+      return new AnsiColorMapper();
+    } catch (Throwable throwable) {
```

Comment:
If my comment on `AnsiColorMapper` is correct, then this should catch `Error`s and not `Throwable`s.